src/Stuart/Repository/JobRepository.php

```diff
@@ -22,6 +23,23 @@ public function __construct($httpClient)
         $this->httpClient = $httpClient;
     }
 
+    /**
+     * @param Job $job
+     *
+     * @return bool|\stdClass
+     */
+    public function validate(Job $job)
```

Comment:
this method should go directly into the `Client` class, I think you should not modify the `JobRepository` within this PR. The `Repository` pattern is confusing, we will get rid of it in another PR.